connectors/couchbase-connector/src/main/scala/zio/connect/couchbase/CouchbaseConnector.scala

```diff
@@ -0,0 +1,53 @@
+package zio.connect.couchbase
+
+import zio.connect.couchbase.CouchbaseConnector.{ContentQueryObject, CouchbaseException, QueryObject}
+import zio.prelude.Subtype
+import zio.stream._
+import zio.{Chunk, Trace}
+
+trait CouchbaseConnector {
```

Comment:
I would add 
def exists(implicit trace: Trace): ZSink[Any, CouchbaseException, QueryObject, QueryObject, Boolean]

and use this in tests also when checking if an item is there.